### ISSUE TITLE
Warning: a promise was created in a handler, but was not returned from it

### ISSUE THREAD

**Issue by preco21** [yagop/node-telegram-bot-api#276]
> Bug Report

* `macOS Sierra 10.12.3`
* `node@7.3.0`
* `node-telegram-bot-api@0.26.0`

### Expected behavior

No warnings.

### Actual behavior

When try to send a message, receive the following warning:

```
(node:91915) Warning: a promise was created in a handler at /Users/preco/Workspace/repos/telegram-bot/node_modules/node-telegram-bot-api/src/telegramPolling.js:102:17 but was not returned from it, see http://goo.gl/rRqMUw
    at new Promise (/Users/preco/Workspace/repos/telegram-bot/node_modules/bluebird/js/release/promise.js:77:14)
```

### Steps to reproduce the Behavior

```javascript
import TelegramBot from 'node-telegram-bot-api';

const bot = new TelegramBot(BOT_TOKEN, {polling: true});

bot.onText(/\/start/, ({chat: {id: chatId}}) => {
  bot.sendMessage(chatId, 'foobar');
});
```

with `NODE_ENV=development` environment variable

**Comment by preco21** [comment]
This seems issue with `request-promise` module.

https://github.com/petkaantonov/bluebird/blob/master/docs/docs/warning-explanations.md#warning-a-promise-was-created-in-a-handler-but-was-not-returned-from-it

I have debug above issue, I could find this error caused on https://github.com/yagop/node-telegram-bot-api/blob/master/src/telegram.js#L155.

**Comment by GochoMugo** [comment]
> If you know what you're doing and don't want to silence all warnings, you can create runaway promises without causing this warning by returning e.g. null:
>
> &mdash; http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-created-in-a-handler-but-was-not-returned-from-it

I believe we know what we are doing, so we can `return null` at [src/telegramPolling.js#L107](https://github.com/yagop/node-telegram-bot-api/blob/release/src/telegramPolling.js#L107).

I smell a PR here! :wink:

**Comment by GochoMugo** [comment]
Just pushed a fix for this in commit [3221d647ccdbcc58562faceabf94ee4c6dae6966](https://github.com/yagop/node-telegram-bot-api/commit/3221d647ccdbcc58562faceabf94ee4c6dae6966). Please try it out and let us know of the results.

**Comment by preco21** [comment]
@GochoMugo Yea! It looks like working now.

**Comment by GochoMugo** [comment]
Fixed in **v0.27.0**.

Thanks @preco21 for your time and effort in fixing this bug.
